style.css

```diff
@@ -9,12 +9,46 @@ html {
   scroll-behavior: smooth;
 }
 
-header {
+.header_container {
   display: flex;
   justify-content: space-between;
   margin: 24px 24px 0 24px;
 }
 
+.nav_link_mobile {
+  font-size: 32px;
+  font-weight: 600;
+  line-height: 44px;
+  color: #3a4a42;
+  text-decoration: none;
+  margin-left: 28px;
+}
+
+.navbar-mobile-close {
+  text-align: right;
+  padding: 10px;
+  color: #67798e;
+}
+
+.navbar-mobile li {
+  margin-bottom: 40px;
+}
+
+.navbar-mobile ul {
+  list-style: none;
+}
+
+.navbar_hidden {
+  display: none;
+}
+
+.navbar-mobile {
+  width: 92%;
+  height: 100%;
+  background-color: white;
+  position: fixed;
+}
```

Comment:
- [ ] The hamburger button and menu should follows the design from the chosen template in Figma (e.g., font, colors, images, tests, margins). 
  :point_right: Your design is messing up with the transparency and overlaying other elements of the page when scrolling down, hence it kills the user experience, when we scroll down some elements are being overlaying the mobile menu, that limits the usability of your site, could you address that issue? Kindly Watch the following screenscast for more context.
:point_right: The mobile menu should fit the whole screen, it shouldn't let the sides (left and right), and bottoms and tops of the page visible, letting them visible is killing the usability of your page, would you make it full-screen please?


https://user-images.githubusercontent.com/56886064/145961603-d993f3fb-eaad-47ca-9771-2530f2ca8dcc.mp4

Reply:
Thank you ! We fix it.